docs/mecsimcalc-library.md

```diff
@@ -653,3 +653,112 @@ Displaying Image
 Downloading Image
 {{ outputs.download }}
 ```
+
+## Quiz Toolkit
+### append_to_google_sheet
+
+[**[Source]**](https://github.com/MecSimCalc/MecSimCalc-utils/blob/v0.1.6/mecsimcalc/quiz_utils.py#L12) 
+
+```python
+append_to_google_sheet(
+    service_account_info = {...},
+    spreadsheet_id = "123abc...",
+    values = [["name", 12837, ...]],
+)
+```
+
+#### Description:
+
+This function appends given values to a specified Google Sheet and optionally includes a current timestamp with each entry. It transforms data into a Google Sheets document, facilitating dynamic data entry directly from your application.
+
+
+#### Arguments:
+
+| Argument               | Type     | Description                                                                                                                                  |
+|------------------------|----------|----------------------------------------------------------------------------------------------------------------------------------------------|
+| **`service_account_info`** | **dict** | The service account credentials used for Google Sheets API authentication.  |
+| **`spreadsheet_id`**       | **str**  | The unique identifier of the target Google Spreadsheet.                     |
+| **`values`**               | **list of lists** | The data to append. Each list element represents a row of data.                                                                              |
+| **`range_name`**           | **str** (optional)  | The A1 notation of the range to start appending data (Defaults to 'Sheet1!A1').                                                              |
+| **`include_timestamp`**    | **bool** (optional) | If True, appends the current timestamp to each row of data (Defaults to True).                                                               |
+
+#### Returns:
+
+| Return Type | Description                              |
+|-------------|------------------------------------------|
+| **`dict`**  | The response from the Google Sheets API, containing details of the append operation. |
+
+#### Example:
+
+#### Code step:
+
+```python
+from datetime import datetime
+import mecsimcalc as msc
+
+def main(inputs):
+    service_account_info = {
+        # Your service account info here
+    }
+    spreadsheet_id = 'your_spreadsheet_id_here'
+    values = [
+    [ inputs['input_1'], inputs['input_2'], inputs['input_3'] ],
+    ]    
+    result = msc.append_to_google_sheet(service_account_info, spreadsheet_id, values, include_timestamp)
```

Comment:
does just writing "include_timestamp" set this to True? The variable isn't defined anywhere.